### DISCUSSION TITLE
Unable to open interferogram

### DISCUSSION THREAD

**Discussion by insarbzh** [alexisInSAR/EZ-InSAR#37] [Help]
Hello，I am unable to open the interferogram on this interface, and there is no response after clicking it.
But the data of this interferogram can be displayed on other computers.
Is there any way to solve it? be deeply grateful!
![图片](https://user-images.githubusercontent.com/127461302/234608556-6289e19d-092e-4503-b398-84344203fb21.png)

**Comment by Matin-GorganSaraj** [comment]
I am sorry i dont know how fix this problem, and i have question (if you answer i we'll be grateful) ; how do you fix the slccreate.m problem i read the solution that alexisInSar gave but i dont know how doing it , and my second question is how i work with ez-insar with local data on my computer? 
With all regard
Matin

**Comment by alexisInSAR** [comment]
Hi all, 

Regarding the display of ifgs, please give me a picture of your terminal to get an idea of what is going on. 

Matin, you can install the new version of EZ-InSAR (2.0.2) to solve this issue. For the local data, you need to create the list and give the correct patch of SLCs, (and orbits) via the button named "Manage data directory". 

Kind regards, 

Alexis

**Comment by insarbzh** [comment]
Sorry, are you referring to the terminal of MATLAB？There is also no response in the  terminal of MATLAB.  But I move the files to another computer that has installed EZ-InSAR and it can be displayed normally.  So it seems like there's no problem with the data.

**Comment by alexisInSAR** [comment]
It is likely to be an error in your installation, if you want to share more information I can investigate. 

Kind regards, 

Alexis
